Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/BasicAdapter.cpp

```diff
@@ -17,7 +17,7 @@ namespace AZ::DocumentPropertyEditor
         NotifyResetDocument();
     }
 
-    Dom::Value BasicAdapter::GenerateContents() const
+    Dom::Value BasicAdapter::GenerateContents()
```

Comment:
curious about this, why isn't this const anymore? Does returning the contents by value actually make the underlying data malleable?

Reply:
Oh forgot to comment on this - `GenerateContents` can include the adapter building callback attributes that might themselves mutate that adapter down the line, so it's not really ergonomic for this to be const. `GetContents` still is treated logically as const though, as this value is cached and you shouldn't have to care as a consumer, but this makes writing subclasses nicer.